js/grades.js

```diff
@@ -222,12 +222,6 @@ var fetchQueue = [];
                         let gradeWrapper = assignment.querySelector(".grade-wrapper");
 
                         let checkbox = document.getElementById("enable-modify");
-                        let editGradeImg = createElement("img", ["grade-edit-indicator"], {
-                            src: chrome.runtime.getURL("imgs/edit-pencil.svg"),
-                            width: 12,
-                            style: `display: ${checkbox && checkbox.checked ? "unset" : "none"};`
-                        });
-                        editGradeImg.dataset.parentId = assignment.dataset.parentId;

```

Comment:
Though I haven't asked @glen3b I don't think we're okay with completely removing the grade edit icons, especially since our users have gotten used to using them over time.

I'm okay with adding click support to the actual grades themselves too, but we need to keep the icon as well

Reply:
I don't really have a preference, I was deferring to you @aopell. That said, for a change like removing the edit icon, we shouldn't make it "just because," there should be some relatively compelling rationale as to why the new version is at least somewhat better, especially given that users are used to the way things are at the moment.

Reply:
Additionally I feel that if we're going to remove the icons (which maybe we will later, I'm not sure) we'd have to do it in such a way that it's obvious that you can click to edit the numbers. Whether that's with `cursor: pointer` or some indication that the number is a textbox I don't know. All I know is we can't remove what everyone is used to yet without a super obvious replacement.

I'm not opposed in general. My school uses canvas, and on canvas you can just click a score to edit without even entering some kind of what-if mode. However, the cursor is a pointer and when you hover over the number a small border appears and a tooltip pops up telling you that you can click to enter a what-if grade. We would need something like that.